### PR TITLE
change export button styling on events/<event-id>/sessions

### LINKED ISSUE #412: non-uniformity of the export button in the app
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
`events/<event-id>/sessions`
image
`events/<event-id>/tickets/attendees`
image
`events/<event-id>/tickets/orders`
image

### PR DESCRIPTION
**Pull request by anu-007**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
change export button styling on events/<event-id>/sessions to maintain uniformity of export button in app

#### Changes proposed in this pull request:

-change export button styling on events/<event-id>/sessions to match styling of export button on events/<event-id>/tickets/orders

demo- https://open-event-frontend-branch3.herokuapp.com/events/ebaf2dbe/sessions

Fixes #412

### CODE REVIEW

File: app/templates/events/view/sessions.hbs [hunk 1]
```diff
@@ -19,16 +19,24 @@
...
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      {{#ui-dropdown class='compact selection'}}
+        <div class="default text">
+          {{t 'Export As'}}
+        </div>
+        <i class="dropdown icon"></i>
         <div class="menu">
-          <div class="item">{{t 'CSV'}}</div>
-          <div class="item">{{t 'PDF'}}</div>
+          <div class="item" data-value="PDF">
```
Comment (niranjan94) [inline app/templates/events/view/sessions.hbs]: put value in lower case

File: app/templates/events/view/sessions.hbs [hunk 2]
```diff
@@ -19,16 +19,24 @@
...
+            <i class="file pdf outline icon"></i>
+            {{t 'PDF'}}
+          </div>
+          <div class="item" data-value="CSV">
```
Comment (niranjan94) [inline app/templates/events/view/sessions.hbs]: put value in lower case

File: app/templates/events/view/sessions.hbs (hunk 1 above)
Comment (anu-007) [inline app/templates/events/view/sessions.hbs]: should I change them on other pages too i.e `events/<event-id>/tickets/orders` and `events/<event-id>/tickets/attendees`

Comment (niranjan94) [inline app/templates/events/view/sessions.hbs]: yep. please do

**Comment by anu-007** [comment]
@geekyd there in the difference in the height of default button(36px) and drop down(38px) 2px more than the height of button due to 1px of border around it, so I wrapped these two buttons inside a div with `ui labeled input ` which removes the border from ui drop-down. please review.thanks !!

**Pull request merged**